src/js/ripple/amount.js

```diff
@@ -176,13 +189,7 @@ Amount.prototype.ratio_human = function(denominator, opts) {
   opts = extend({ }, opts);
 
   var numerator = this;
-
-  if (typeof denominator === 'number' && parseInt(denominator, 10) === denominator) {
-    // Special handling of integer arguments
-    denominator = Amount.from_json(String(denominator) + '.0');
-  } else {
-    denominator = Amount.from_json(denominator);
-  }
+  denominator = Amount.from_json(noIntegers(denominator));
```

Comment:
> Perhaps an IOU style amount should be created for these cases where noIntegers is used, otherwise the number is parsed as a native amount

I think the code is trying to allow the denominator parameter to be a native amount, but forcing it to be interpreted as XRP rather than drops, as it would be without the integer check. This can also be fixed by https://ripplelabs.atlassian.net/browse/RLJS-215